pipeline.py

```diff
@@ -28,29 +34,43 @@ class MorphologicalToken:
     Stores language params for each processed token
     """
     def __init__(self, original_word, normalized_form):
-        pass
+        self.original_word = original_word
+        self.normalized_form = normalized_form
+        self.mystem_tags = ''
+        self.pymorphy_tags = ''
 
     def __str__(self):
-        return "MorphologicalToken instance here"
+        return f'{self.normalized_form}<{self.mystem_tags}>({self.pymorphy_tags})'
+
+    def public_method(self):
```

Comment:
You try to fix lint by using unnecessary stubs. Actually, it would be better to change lint params. In that case lint check would not pay any attention to "to few public methods" case. And you would not have to change given interface. If you want, you may fix this one. Tip: pylint settings are in .pylintrc files:)

Reply:
Thank you for the tip) I've fixed this error.